website/docs/index.html.markdown

```diff
@@ -79,6 +79,63 @@ provider "mysql" {
 }
 ```
 
+TLS Configuration can also be sourced from secrets such as Google Secret Manager or variables like so:
```

Comment:
This does not say a lot. Anything can be in that secrets.

What I imagine: when I open
https://registry.terraform.io/providers/petoju/mysql/latest/docs#ca_cert
(sourced from here), I will see, that ca_cert can be a filename or a text certificate starting with `-----BEGIN`. Similar with the key.

Once anyone knowing terraform spots that those may be just file contents, he can easily use google secrets manager or basically anything.

Reply:
Np, this looks better!